tests/test_driver_out.py

```diff
@@ -16,6 +16,7 @@ def test_driver_out(self):
         params.nV = 50  # vehicles
         params.simTime = 4
         params.nD = 1
+        params.user_controlled_vehicles_count = 0
```

Comment:
no i tego bardzo chcialbym uniknac.

Reply:
Jeśli uda mi się zaimplementować przekazywanie tego parametra w inny sposób niż robię to obecnie, to ta konstrukcja powinna stąd zniknąć.